pages/_app.js

```diff
@@ -121,6 +91,19 @@ export default function App({ Component, pageProps }) {
     return () => clearInterval(interval);
   }, [isActive3]);
 
+  useEffect(() => {
+    setTrys(trys);
+    if (checkFruits) {
+      tryText;
+      save();
+
+      setTrys(1);
+    }
+    if (!checkFruits && !isActive3 && !isActive1 && !isActive2) {
+      setTrys(trys + 1);
+    }
+  }, [isActive3 || isActive2 || isActive1]);
```

Comment:
stilling missing dependecies:

<img width="603" alt="image" src="https://user-images.githubusercontent.com/115098194/232736398-96df1dd9-c1d1-4138-b1cc-b52e4f5bad6b.png">

whereever possible, try to code without sideeffects. so try to avoid useEffect, whereever possible as you run into infinite loops very fast.

you can handle the setTries per onClick event on button. no need to use a side-effect here. 

see the codesandbox, i've sent you per slack for further inspiriation.